code/__DEFINES/sound.dm

```diff
@@ -1,3 +1,4 @@
 //max channel is 1024. Only go lower from here, because byond tends to pick the first availiable channel to play sounds on
 #define CHANNEL_LOBBYMUSIC 1024
 #define CHANNEL_ADMIN 1023
+#define CHANNEL_VOX 1022
```

Comment:
Granted, that comment was me theory crafting. I know the 0 channel STARTS at 1 but I don't know if it resets to 1 after the sound ends. I'll implement this, it's more reliable